board/raspberrypi/post-build.sh

```diff
@@ -30,3 +30,8 @@ if [ -f "${BOARD_DIR}/osmc-devinput-remote.json" ]; then
 	mkdir -p "${TARGET_DIR}/usr/share/WPEFramework/RemoteControl/"
 	cp -pf "${BOARD_DIR}/osmc-devinput-remote.json" "${TARGET_DIR}/usr/share/WPEFramework/RemoteControl/devinput-remote.json"
 fi
+
+if  [ -e "${TARGET_DIR}/etc/WPEFramework/plugins/BluetoothRemoteControl.json" ] && [ -f "${BOARD_DIR}/bluetooth-remote.json" ]; then
```

Comment:
Please move this to the CMakeList.txt otherwise we will break Yocto. Part of the Bluetooth RemoteControl plugin

Reply:
Removed.